compiler/luci-interpreter/src/core/RuntimeGraph.cpp

```diff
@@ -29,15 +29,22 @@ class RuntimeGraph::TensorAllocPlan
   std::vector<std::vector<Tensor *>> _alloc_plan;
   std::vector<std::vector<Tensor *>> _dealloc_plan;
   bool _valid = false;
+  IMemoryManager *_memory_manager;
```

Comment:
Q. may be const?

Reply:
I think that in the future we may have some kind of memory manager that can change, for example, collects some statistics. And that is why it can be better if we remain this _memory_manager as not constant ptr.